test/templateupdaterequest/templateupdaterequest.go

```diff
@@ -62,10 +68,43 @@ var _ Option = Complete("")
 
 func (c Complete) applyToTemplateUpdateRequest(r *toolchainv1alpha1.TemplateUpdateRequest) {
 	if r.Name == string(c) {
+		r.Status.Conditions = []toolchainv1alpha1.Condition{
+			{
+				Type:               toolchainv1alpha1.TemplateUpdateRequestComplete,
+				Status:             corev1.ConditionTrue,
+				LastTransitionTime: metav1.NewTime(time.Now()),
+			},
+		}
+	}
+}
+
+// CompleteSince sets the status condition to "complete" on the TemplateUpdateRequest, using the value as an offset to time.Now
+type CompleteSince time.Duration
```

Comment:
My English is not good, so correct me if I'm wrong - in case of `CompleteSince` I would expect that we should pass a date/time since the condition has been completed. However, this expects duration for how long the condition is completed, so shouldn't it be `CompletedBefore`?

Reply:
actually, it should be `CompleteFor` since it's a duration and not a time...

Reply:
yes, `CompleteFor` is much better :+1:

Reply:
done in f3cf7a4